warehouse/ingest-configuration/pom.xml

```diff
@@ -18,6 +18,11 @@
             <artifactId>javax.mail-api</artifactId>
             <version>1.5.4</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.hadoop</groupId>
+            <artifactId>hadoop-common</artifactId>
+            <version>3.3.3</version>
```

Comment:
Use `${verison.hadoop}`

Reply:
+1, but you don't even need to do that.  It's managed in the parent pom, so just remove the version.

Reply:
removed.